src/Microsoft.Data/StringColumn.cs

```diff
@@ -0,0 +1,107 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.Data
+{
+    public partial class StringColumn : BaseColumn
+    {
+        internal IList<IList<string>> stringBuffers = new List<IList<string>>(); // To store more than intMax number of strings
```

Comment:
why is this field internal? Typically fields should be private, and if they need to be exposed, expose them through a property.

Reply:
Changed